### PR TITLE
Update commands

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) incorporates comments Oleg left in https://jira.corp.adobe.com/browse/CEXT-652

## Affected pages

-  ...

## Links to related PRs or Jira tickets

-  ...

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My changes follow the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/events/configure-commerce.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ description: Learn how to install the Commerce modules needed for Adobe I/O Even
 
 After you have created an [App Builder project](./project-setup.md), you must install the Commerce modules that enable integrations with Adobe I/O Events.
 
+<!-- This info will change as we get closer to GA. The modules will install autmatically on Cloud, while on-premises customers will get the modules from Marketplace. -->
```
Comment (jhadobe) [inline src/pages/events/configure-commerce.md:10]:
```suggestion
<!-- This info will change as we get closer to GA. The modules will install automatically on Cloud, while on-premises customers will get the modules from Marketplace. -->
```
Trailing space causing linter error.

Comment (oshmyheliuk) [inline src/pages/events/configure-commerce.md:10]: * automatically

**Review by jhadobe — CHANGES_REQUESTED** [review]

File: src/pages/events/commands.md [hunk 2]
```diff
@@ -30,18 +30,7 @@ Adobe Commerce provides the following commands to configure and process events:
...
-
-```json
-{
- "provider": {
-     "label": "My Adobe Commerce Events",
-     "description": "Provides out-of-process extensibility for Adobe Commerce"
-     }
- }
-```
+The `--label` and `--description` arguments are optional but recommended.
```
Comment (oshmyheliuk) [inline src/pages/events/commands.md:33]: Why the previous description was changed to this? If arguments are not provided then `event-types.json` file should be created, nothing have changed in the command behavior

Comment (keharper) [reply src/pages/events/commands.md:33]: Oleg said don't mention `event-types.json`. If I can't mention it, I have to be more insistent that the arguments be specified at runtime.

Comment (oshmyheliuk) [reply src/pages/events/commands.md:33]: So in this case arguments are required (not optional) if they are not provided command will be looking for the file.

**Review by jhadobe — APPROVED** [review]

File: src/pages/events/commands.md (hunk 2 above)
Comment (keharper) [reply src/pages/events/commands.md:33]: Reverted these changes.

**Pull request merged**